roles/shibboleth/tasks/main.yml

```diff
@@ -28,6 +28,7 @@
       - "/opt/shibboleth-nginx/log:/var/log"
       - "/opt/shibboleth-nginx/shibboleth-nginx-default.conf:/etc/nginx/conf.d/default.conf"
       - "/opt/shibboleth-nginx/shibboleth2.xml:/etc/shibboleth/shibboleth2.xml"
+      - "/opt/shibboleth-nginx/shibboleth-spid/main.css:/etc/shibboleth/main.css"
```

Comment:
In realtà quell'alias [l'ho commentato](https://github.com/italia/spid-sp-playbook/blob/master/roles/shibboleth/templates/shibboleth-nginx-default.conf.j2#L33) perchè è malvagio: espone tutta la directory di configurazione, compresi i certificati che ci sono dentro 😞 

Quindi se vogliamo tenere i css e gli html in `/etc/shibboleth` dobbiamo esporli con degli alias specifici per ogni file.